btree/btree.c

```diff
@@ -2921,6 +2925,493 @@ static void ff_capture(struct slot *slot, struct m0_be_tx *tx)
 	M0_BTREE_TX_CAPTURE(tx, seg, h, hsize);
 }
 
+/**
+ *  --------------------------------------------
+ *  Section END - Fixed Format Node Structure
+ *  --------------------------------------------
+ */
+
+/**
+ *  --------------------------------------------
+ *  Section START -
+ *  Variable Sized Key and Values Node Structure
+ *  --------------------------------------------
+ *
+ * Proposed node structure
+ *
+ *
+ *                        +--------------------------+
+ *                        |                          |
+ *                 +------+-----------------------+  |
+ *                 |      |                       |  |
+ *             +---+------+-------------------+   |  |
+ *             |   |      |                   |   |  | DIR                    16   11       3  0
+ * +----------+v-+-v--+---v----+-------------++-+-++-++--+--+--+--------------+----+--------+--+
```

Comment:
The tip of the arrows from the DIR to the Key list or Value list is not visible. Can you add another line which would have the arrow tips by themselves as the arrow direction here is important.

Reply:
Updated